projects/demo/src/modules/components/editor/custom-tool/color-picker/examples/1/index.html

```diff
@@ -0,0 +1,15 @@
+<tui-editor
+    class="editor"
+    [formControl]="control"
+    [tools]="builtInTools"
+>
+    <ng-container ngProjectAs="color">
+        <!-- Also you can use any custom component here -->
+        <tui-text-full-color-picker></tui-text-full-color-picker>
+    </ng-container>
+
+    <ng-container ngProjectAs="hilite">
+        <!-- Also you can use any custom component here -->
+        <tui-highlight-full-color-picker></tui-highlight-full-color-picker>
```

Comment:
What is the difference between `<tui-text-full-color-picker />` and `<tui-highlight-full-color-picker />` ?
Only icon and some texts ?

Should we create universal `<tui-full-color-picker />` with configurable params ?

Reply:
fixed